install: add pre-commit config (#1059)
iterative/dvc.org: public/static/docs/command-reference/install.md

To disable them, you need to **remove** or **edit** those files (i.e.
`rm .git/hooks/post-checkout`, `vim .git/hooks/pre-commit`).

> ## Using Pre-Commit tool

Reviewer: ```suggestion
## Using the **pre-commit** tool
```
No need for upper case here, I think. Maybe just bold

DVC provides support for [pre-commit](https://pre-commit.com/) users. To adjust

> `.pre-commit-config.yaml` you could either use

Reviewer: ```suggestion
DVC provides support to manage Git hooks with [pre-commit](https://pre-commit.com/).
To adjust `.pre-commit-config.yaml` you can either use
```

`dvc install --use-pre-commit-tool`, or add these entries by hand:

```yaml
repos:
- hooks:
  - id: dvc-pre-commit
    language_version: python3
    stages:
    - commit
  - id: dvc-pre-push
    language_version: python3
    stages:
    - push
  - always_run: true
    id: dvc-post-checkout
    language_version: python3
    stages:
    - post-checkout
  repo: https://github.com/iterative/dvc
  rev: master
```

## Options

- `--use-pre-commit-tool` - install pre-commit/pre-push/post-checkout hooks
  into the [pre-commit](https://pre-commit.com/) config

>   (`.pre-commit-config.yaml`).

Reviewer: ```suggestion
- `--use-pre-commit-tool` - installs pre-commit, pre-push, post-checkout Git
  hooks into the [pre-commit](https://pre-commit.com/) config file
  (`.pre-commit-config.yaml`).
```
